variantlib/models/variant.py

```diff
@@ -194,11 +193,23 @@ def __post_init__(self) -> None:
         # Execute the validator
         super().__post_init__()
 
+    @property
+    def is_null_variant(self) -> bool:
```

Comment:
By the way, I really dislike the idea of starting property name with questions like "is". Questions are active part of language, so they naturally suggest a method.

Reply:
OK for me - change to methods